packages/renderer/src/lib/network/NetworksList.svelte

```diff
@@ -111,6 +116,13 @@ function key(network: NetworkInfoUI): string {
 
 <NavPage bind:searchTerm={searchTerm} title="networks">
 
+  {#snippet additionalActions()}
+    {#if providerConnections.length > 0}
+      <Button on:click={gotoCreateNetwork} icon={faPlusCircle} title="Create a volume" aria-label="Create"
```

Comment:
```suggestion
      <Button onclick={gotoCreateNetwork} icon={faPlusCircle} title="Create a network" aria-label="Create"
```

- we can use Svelte5 synthax `onclick` directly
- we also need to rename to network